External Task Client can not be built

Hi, I've cloned the repo but could not run it on my computer, especially the external task client.

after `npm install`
and `node service.js`
I got this output:

```
internal/modules/cjs/loader.js:985
  throw err;
  ^

Error: Cannot find module 'node-fetch'
Require stack:
- C:\Users\Ingo Richtsmeier\git\Make-Rest-Calls-From-Camunda-Example\SearchContributorService\service.js
←[90m    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:982:15)←[39m
←[90m    at Function.Module._load (internal/modules/cjs/loader.js:864:27)←[39m
←[90m    at Module.require (internal/modules/cjs/loader.js:1044:19)←[39m
←[90m    at require (internal/modules/cjs/helpers.js:77:18)←[39m
    at Object.<anonymous> (C:\Users\Ingo Richtsmeier\git\Make-Rest-Calls-From-Camunda-Example\SearchContributorService\service.js:1:15)
←[90m    at Module._compile (internal/modules/cjs/loader.js:1158:30)←[39m
←[90m    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)←[39m
←[90m    at Module.load (internal/modules/cjs/loader.js:1002:32)←[39m
←[90m    at Function.Module._load (internal/modules/cjs/loader.js:901:14)←[39m
←[90m    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)←[39m {
  code: ←[32m'MODULE_NOT_FOUND'←[39m,
  requireStack: [
    ←[32m'C:\\Users\\Ingo Richtsmeier\\git\\Make-Rest-Calls-From-Camunda-Example\\SearchContributorService\\service.js'←[39m
  ]
}
```
I saw that the `package.json` is missing in the repo and I added the one from the camunda-external-task-client-js. But I missend some libraries (and the version).

Could you please commit the `package.json` to this repo? Thank you.

--- a/SearchContributorService/package.json
+++ b/SearchContributorService/package.json
@@ -1,0 +1,16 @@
+{
+  "name": "searchcontributorservice",
+  "version": "1.0.0",
+  "description": "",
+  "main": "service.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "camunda-external-task-client-js": "^2.1.0",
+    "node-fetch": "^2.6.1"
+  }
+}

